app/views/site/_study_visualize.html.erb

```diff
@@ -56,6 +56,14 @@
 </div>
 <script type="text/javascript" nonce="<%= content_security_policy_script_nonce %>">
   <% if @study.can_visualize_clusters? %>
+    if (typeof window.SCP.pageIsFullyInteractive === 'undefined') {
+      window.SCP.startPendingEvent('user-action:page:view:site-study',
+                                 {},
+                                 'plot:',
+                                 true);
+    }
+    window.SCP.pageIsFullyInteractive = true;
```

Comment:
I'm not sure I understand this variable name -- how do we know the page is fully interactive at this point?  the data hasn't rendered yet

Is what we're looking for 'initialPageLoadDone'?

Reply:
Good point.  Neither name is quite right.  In 4d89e91, I renamed it to `pushedPendingPageViewEvent`.